src/docs/Product/getRequesters.yaml

```diff
@@ -0,0 +1,36 @@
+paths:
+  /api/product/{productId}/requesters:
+    get:
+      tags:
+        - Product
+      summary: Shows IDs of all requesters of a product (Must be logged in)
+      parameters:
+        - in: path
+          name: productId
+          schema:
+            type: string
+          required: true
+          description: 'ID of the product'
+          examples:
+            'real ID of a deleted product':
+              value: '62ee5c280262c62f56124dfd'
+            'some fake ObjectId':
```

Comment:
instead of a fake id maybe we can add one of the product id from another user to see the not authorized error